### PR TITLE
sci-geosciences/grass: version bump 7.8.5

### PR DESCRIPTION
**Pull request by hamarituc**
version bump

### CODE REVIEW

File: sci-geosciences/grass/grass-7.8.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+		addpredict "${mesa_cards}"
+	fi
+	local ati_cards=$(echo -n /dev/ati/card* | sed 's/ /:/g')
+	if test -n "${ati_cards}"; then
+		addpredict "${ati_cards}"
+	fi
+	shopt -u nullglob
+	addpredict /dev/nvidiactl
+
+	eapply_user
```
Comment (a17r) [inline sci-geosciences/grass/grass-7.8.5.ebuild:138]: This is redundant after you already called `default`.

**Review by a17r — CHANGES_REQUESTED** [review]
Maybe you want to drop oldest version in a second commit?

**Comment by hamarituc** [comment]
> Maybe you want to drop oldest version in a second commit?

I dropped version 7.8.3, but kept 7.8.4 and requested stabilization for the latter version instead. I hope it's OK.

**Review by juippis — APPROVED** [review]
With this amount of seds I'm surprised it isn't broken.

**Comment by a17r** [comment]
Bump apparently happened in commit 8f11c6038f858cc6ae60df12ee37f0bc0b462fb5. Thanks!

**Pull request closed without merging**